src/main/java/com/epam/productsreview/config/ApplicationConfig.java

```diff
@@ -1,13 +1,25 @@
 package com.epam.productsreview.config;
 
+import liquibase.integration.spring.SpringLiquibase;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import javax.sql.DataSource;
+
 /**
  * Created by saurabh on 26/2/19.
  */
 @Configuration
 public class ApplicationConfig {
-
-
+    @Autowired
+    private DataSource dataSource;
+    @Bean
+    public SpringLiquibase liquibase() {
+        SpringLiquibase liquibase = new SpringLiquibase();
+        liquibase.setChangeLog("classpath:liquibase-changeLog.xml");
```

Comment:
Need to point liquibase-outputChangeLog.xml file?

Reply:
removed it as this was only to test the behaviour